.github/workflows/container-autoupdate.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   push-container:
     name: Push container to Quay.io
     runs-on: ubuntu-latest
+    needs: ["unit-tests", "rpm-tests"]
```

Comment:
rpm-tests uses a separate container, no need to depend on that.

Reply:
Good point! This was just a test and as you can see above I have to merge workflow files first to make this work.